woo-components/src/components/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,48 @@
+const React = require('react');
+const { PropTypes } = React;
+
+class Pagination extends React.Component {
+
+  render () {
+    const PageLinks = ({amount}) => {
+      return (
+        <div>
+          {
+            Array(amount).fill().map((_, i) => {
+              return (
+                <button className='btn btn-default' key={i}
+                  onClick={() => this.props.onPagination(i * this.props.step, this.props.step)}>
+                  {i + 1}
+                </button>
+              );
+            })
+          }
+        </div>);
+    };
+
+    const pageCount = (
+      Math.floor(this.props.total / this.props.step) +
+      Math.min(this.props.total % this.props.step, 1)
+    );
+
+    return (
+      <div className='woo-pagination'>
+        <PageLinks amount={pageCount} />
+      </div>
+    );
+  }
+}
+
+Pagination.defaultProps = {
+  first: 0,
+  step: 10
+};
+
+Pagination.propTypes = {
```

Comment:
Yes I wouldn't count on amount of total items because that might be an entirely separate query. Also if you have cursors, you can't calculate pages that easily besides keeping state of number of number of cursors you used. I know that middlelayer project list pagination for example uses cursors. We don't have to add it know but it would be nice to keep this in mind.